addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -97,7 +71,7 @@ export class ColorPalette extends Component {
         this.init();
         onWillStart(async () => {
             if (this.props.getTemplate) {
-                await ColorPalette.loadDependencies(this.props.getTemplate);
+                this.colorpickerArch = await this.props.getTemplate();
```

Comment:
Ouch, I thought I had you rename all those "arch" things to "template"... looks like we missed some of them... :/